src/gui.cc

```diff
@@ -149,7 +149,8 @@ error GUI::DisplayError(const error &err) {
         ss << "You are offline (" << err << ")";
         if (kBackendIsDownError == err) {
             DisplayOnlineState(kOnlineStateBackendDown);
-        } else {
+        }
+        else if (kRateLimit != err) {
```

Comment:
`IsNetworkingError` for 429 returns `false` as far as I understood this, so `err` should never be `kRateLimit` at this point and we don't need this check. Am I correct? @NghiaTranUIT